azure-pipelines/build/templates/doclient-lite-docker-steps.yml

```diff
@@ -2,18 +2,46 @@
 # Consume this steps template in one or more jobs by passing in parameter values.
 
 parameters:
-- name: targetOsArch    # example: debian10-arm32
+- name: targetOS        # example: ubuntu1804
   type: string
-- name: dockerImageName # example: jimsonmsft/debian10-arm32:latest
+- name: targetOsArch    # example: ubuntu1804-x64
   type: string
 - name: config          # debug/release/minsizerel/relwithdebuginfo
   type: string
 
 steps:
+- task: Docker@2
```

Comment:
I think all the steps from login to install & register qemu can be moved to a new template and referenced from do-docker-jobs.yml.

Not urgent, feel free to open a task.

Reply:
Well, the registering of QEMU will happen during image provisioning (once this script is uploaded as an artifact to Image Factory), which leaves just the login and pull steps, which could be templatized further, but at that point I'm not sure if it's worth it for two tasks

Reply:
Agreed, may not be worth it for just a couple of steps.